src/ng-select/lib/ng-select.component.spec.ts

```diff
@@ -3035,7 +3035,7 @@ describe('NgSelectComponent', () => {
         describe('edit search query', () => {
             it('should allow edit search if option selected & input focused', fakeAsync(() => {
                 const fixture = createTestingModule(
-                    NgSelectTestCmp,
+                    NgSelectTestComponent,
```

Comment:
I was able to fix https://github.com/ng-select/ng-select/pull/1996#issuecomment-969047202

by adding a `tick(200)` before line 3054 (which I can't comment on in this diff). I'm not sure why the compontent is taking more time to resolve though :thinking:

Reply:
Interesting. For me I had to add `tick(2000)` not `tick(200)`. Don't why.

Reply:
> I was able to fix [#1996 (comment)](https://github.com/ng-select/ng-select/pull/1996#issuecomment-969047202)
> 
> by adding a `tick(200)` before line 3054 (which I can't comment on in this diff). I'm not sure why the compontent is taking more time to resolve though 🤔

This test seems to be flakey. Adding `tick(2000)` sometimes passes and fails other times.